example/ssl-preaccept/ssl-preaccept.cc

```diff
@@ -43,7 +43,7 @@ namespace
 {
 std::string ConfigPath;
 typedef std::pair<IpAddr, IpAddr> IpRange;
-typedef std::deque<IpRange> IpRangeQueue;
+using IpRangeQueue = std::deque<IpRange>;
```

Comment:
clang-tidy isn't 100%.  There will need to be a followup commit to fix the remaining items.